app/src/main/java/com/client/fire_and_water/CheckInFragment.kt

```diff
@@ -0,0 +1,39 @@
+package com.client.fire_and_water
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.navigation.fragment.findNavController
+import com.client.fire_and_water.databinding.FragmentCheckInBinding
+
+class CheckInFragment : Fragment() {
+    private var _binding: FragmentCheckInBinding? = null
+
+    // This property is only valid between onCreateView and
+    // onDestroyView.
+    private val binding get() = _binding!!
+
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        _binding = FragmentCheckInBinding.inflate(inflater, container, false)
```

Comment:
Does `FragmentCheckInBinding` have a parameter check for null?

Reply:
Are you writing about container?
It`s `@nullable` parameter in function description

Reply:
Okay